common/src/main/java/com/microsoft/identity/common/internal/providers/microsoft/azureactivedirectory/AzureActiveDirectoryAuthorizationResult.java

```diff
@@ -22,7 +22,20 @@
 // THE SOFTWARE.
 package com.microsoft.identity.common.internal.providers.microsoft.azureactivedirectory;
 
-import com.microsoft.identity.common.internal.providers.oauth2.AuthorizationResult;
+import com.microsoft.identity.common.internal.providers.microsoft.MicrosoftAuthorizationResult;
+import com.microsoft.identity.common.internal.providers.oauth2.AuthorizationErrorResponse;
+import com.microsoft.identity.common.internal.providers.oauth2.AuthorizationStatus;
 
-public class AzureActiveDirectoryAuthorizationResult extends AuthorizationResult {
+public class AzureActiveDirectoryAuthorizationResult extends MicrosoftAuthorizationResult {
+
+
+    public AzureActiveDirectoryAuthorizationResult(final AuthorizationStatus authStatus, final AzureActiveDirectoryAuthorizationResponse authResponse) {
```

Comment:
This constructor implementation is the same as `MicrosoftStsAuthorizationResult` -- consider pulling this up into `MicrosoftAuthorizationResult`

Reply:
Good catch and suggestion, moved it super class.